source/enterprise/tech-guide/update-eca-guide.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: Update the Client Administration
+github_link: enterprise/tech-guide/update-eca-guide.md
+indexed: false
+---
+
+This guide describes how you update the Enterprise Client Administration.
+
+<div class="toc-list"></div>
+
+## How to update the ECA
+
+### 1. Backup the ECA.
+
+First of all, you have to[back up](/enterprise/tech-guide/backup-eca-guide/)the ECA.
+
+For this usecase we already provide an userguide. [Back up the eca](/enterprise/tech-guide/backup-eca-guide/)
+
+----------------------
+
+### 2. Install the ECA
+
+After you have created a backup of your ECA, you can install the new version.
+
+This is very easy.
+
+Just [download](/enterprise/tech-guide/installation-guide/) the updated version of the ECA and and then extract all files into your existing directory.
```

Comment:
the ECA Update package (sorry my mistake)

extract all files into your existing eca root directory